docs/core/linux-prerequisites.md

```diff
@@ -122,6 +122,15 @@ For versions earlier than .NET Core 2.1, following dependencies are also require
 * libunwind8
 * libuuid1
 
+For .NET Core applications that use the *System.Drawing.Common* assembly, you also need the following dependency:
+
+* libgdiplus (version 6.0.1 or later) - required by applications that use the *System.Drawing.Common* assembly.
```

Comment:
That suggestion was an alternative version in case we wanted to combine this entry into the first list. If it's going to stay here, then we don't need to say it twice. Hopefully, it makes sense.

```suggestion
* libgdiplus (version 6.0.1 or later)
```

Reply:
Ah, makes sense 🤦‍♂ . Fixed it!